Commons/bukkit/src/main/java/tc/oc/commons/bukkit/report/ReportFormatter.java

```diff
@@ -1,12 +1,14 @@
 package tc.oc.commons.bukkit.report;
```

Comment:
Your changes in reportformatter don’t seem to do anything. They can probably just be discarded.